### ISSUE TITLE
parity_nodeKind: -32601: Method not found

### ISSUE THREAD

**Issue by ruchevits** [paritytech/parity#5467]
Trying to run the example from `https://github.com/paritytech/oo7-parity`

Parity is running with the following arguments:
```bash
parity -l all \
       --mode active \
       --chain $CHAINS_DIR/$CHAIN \
       --db-path $DB_PATH \
       --keys-path $DB_PATH/keys \
       --identity $IDENTITY \
       --unlock $COINBASE \
       --password $CONFIG_DIR/password \
       --no-discovery \
       --reserved-peers $CONFIG_DIR/peers.enodes \
       --jsonrpc-port 8545 \
       --jsonrpc-interface "0.0.0.0" \
       --jsonrpc-apis "web3,eth,net,parity,parity_accounts,traces,rpc,personal" \
       --jsonrpc-hosts "all" \
       --author $COINBASE \
       --reseal-min-period 10 \
       --dapps-interface "0.0.0.0" \
       --dapps-hosts "all" \
       --dapps-port "8080" \
       --dapps-user "user" \
       --dapps-pass "123" \
       --ui-interface "0.0.0.0" \
       --ui-port "8180" \
       $*
```

**Comment by 5chdn** [comment]
Which version are you running? nodekind was introduced in #5025 (for #5007) and is only available in 1.7.0 nightly branch.

**Comment by ruchevits** [comment]
Oh, got it, I was running 1.6.6. Thank you for the quick answer!
